packages/react-ai/src/components/AIConversation/views/Controls/AvatarControl.tsx

```diff
@@ -4,40 +4,42 @@ import { withBaseElementProps } from '@aws-amplify/ui-react-core/elements';
 import { ConversationMessage } from '../../types';
 import { AvatarsContext } from '../../context';
 import { AIConversationElements } from '../../context/elements';
-const { Span, Text, View } = AIConversationElements;
+const { Icon, Span, Text, View } = AIConversationElements;
 
 const AVATAR_BLOCK = 'ai-avatar';
 
-const iconAttributes = {
-  'aria-hidden': true,
-  className: `${AVATAR_BLOCK}__icon`,
-  xmlns: 'http://www.w3.org/2000/svg',
-  height: '100%',
-  width: '100%',
-};
+const DEFAULT_USER_ICON = withBaseElementProps(Icon, {
+  variant: 'user-avatar',
+});
 
-const avatarIconProps = () => ({
-  children: (
-    <svg viewBox="0 0 24 24">
+const DEFAULT_AI_ICON = () => (
```

Comment:
The Raven icon is custom so not including it as a default bc it would require overriding almost every attribute but curious if anyone has opinions on that

Reply:
By custom do you mean it doesnt share most of it's attributes with the other icons? Why is that an issue?

Reply:
Yeah I just meant it's not an icon you could get from material icons. I was thinking using the IconElement and overriding all the default props would probably be more verbose than just hard coding the svg but curious if we do want to use that util here instead.